<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSettingSignalWindowPropertyDlg</class>
 <widget class="QDialog" name="BCSettingSignalWindowPropertyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>窗口标题</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>标题：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pTitleLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>窗口坐标</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pPoxXLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pPoxYLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>窗口尺寸</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>W</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pWidthLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pHeightLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>64</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="m_pOkBtn">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="m_pCancelBtn">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
